api/src/main/java/org/apache/cloudstack/api/response/NetworkResponse.java

```diff
@@ -163,6 +163,10 @@ public class NetworkResponse extends BaseResponseWithAssociatedNetwork implement
     @Param(description = "the domain name of the network owner")
     private String domain;
 
+    @SerializedName(ApiConstants.DOMAIN_PATH)
+    @Param(description = "path of the Domain the network belongs to", since = "4.18.1")
```

Comment:
@pierrelefevre 
the target branch "main" is 4.19.0.0, not 4.18.1

if you want it to be merged into 4.18.1.0, please rebase with 4.18 branch and change target branch to 4.18. thanks